"""Fixed bug in models after previous update

Revision ID: c7266b53c0e3
Revises: 4691b8df1635
Create Date: 2023-04-10 21:14:00.259886

"""
from alembic import op
import sqlalchemy as sa


# revision identifiers, used by Alembic.
revision = 'c7266b53c0e3'
down_revision = '4691b8df1635'
branch_labels = None
depends_on = None


def upgrade() -> None:
    # ### commands auto generated by Alembic - please adjust! ###
    op.add_column('advertisement_logs', sa.Column('parenting_tip_id', sa.Integer(), nullable=False))
    op.drop_constraint(None, 'advertisement_logs', type_='foreignkey')
    op.create_foreign_key(None, 'advertisement_logs', 'parenting_tips', ['parenting_tip_id'], ['id'])
    op.create_foreign_key(None, 'advertisement_logs', 'advertisements', ['ad_id'], ['id'])
    op.drop_column('advertisement_logs', 'tip_id')
    op.drop_column('parenting_tips', 'advertisement')
    # ### end Alembic commands ###


def downgrade() -> None:
    # ### commands auto generated by Alembic - please adjust! ###
    op.add_column('parenting_tips', sa.Column('advertisement', sa.VARCHAR(), nullable=True))
    op.add_column('advertisement_logs', sa.Column('tip_id', sa.INTEGER(), nullable=False))
    op.drop_constraint(None, 'advertisement_logs', type_='foreignkey')
    op.drop_constraint(None, 'advertisement_logs', type_='foreignkey')
    op.create_foreign_key(None, 'advertisement_logs', 'parenting_tips', ['tip_id'], ['id'])
    op.drop_column('advertisement_logs', 'parenting_tip_id')
    # ### end Alembic commands ###
